thingsvision/core/extraction/extractors.py

```diff
@@ -350,15 +350,14 @@ def __init__(
             device=device,
         )
 
-    def _download_and_save_model(self, model_url: str,
-                                 output_model_filepath: str, unique_model_id: str):
+    def _load_vissl_state_dict(self, model_url: str, unique_model_filename: str):
         """
         Downloads the model in vissl format, converts it to torchvision format and
```

Comment:
Maybe adapt docstring and write that `load_state_dict_from_url` is using a cached version if available.

Reply:
I adapted the docstring, please have a look if that's clear now!